### PR TITLE
Add NAB - NaBeeld

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN

### CODE REVIEW

**Comment by palemieux** [comment]
@japierce1028 Isn't _NAB_ potentially confusing with the organization of the same name?

**Review by palemieux — CHANGES_REQUESTED** [review]
Isn't the name confusing with https://www.nab.org/ ?

**Comment by japierce1028** [comment]
> Isn't the name confusing with https://www.nab.org/ ?

Sure, but we don't have any policy for possible conflicts. I'll bet there are other common names out there. IMF is a valid code, for instance. Not to mention WTF. I would let them have it, but if others object I'm happy to ask for a new one.

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -2795,6 +2795,16 @@
...
   },
+  {
+    "code": "NAB",
+    "contact": {
+      "address": "Harentstraat 17, 2820 Bonheiden, Belgium",
+      "email": "june@nabeeld.be",
+      "name": "June Verelst"
+    },
+    "description": "NaBeeld",
+    "url": "http://www.nabeeld.be"
```
Comment (palemieux) [inline src/main/data/facilities.json:2806]:
```suggestion
    "url": "https://www.nabeeld.be"
```

**Review by palemieux — APPROVED** [review]

**Pull request merged**